examples/src/main/scala/org/apache/spark/examples/ml/LDAExample.scala

```diff
@@ -55,19 +55,21 @@ object LDAExample {
       .setMaxIter(10)
       .setFeaturesCol(FEATURES_COL)
     val model = lda.fit(dataset)
-    val transformed = model.transform(dataset)
 
     val ll = model.logLikelihood(dataset)
     val lp = model.logPerplexity(dataset)
+    println(ll)
+    println(lp)
```

Comment:
println(s"The lower bound on the log likelihood of the entire corpus: $ll")
println(s"The upper bound bound on perplexity: $lp")